modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -25,6 +25,11 @@ ifdef::openshift-dedicated,openshift-rosa[]
 endif::openshift-dedicated,openshift-rosa[]
 * You have installed the OpenShift CLI (`oc`).
 * You have set up a remote write compatible endpoint (such as Thanos) and know the endpoint URL. See the link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Prometheus remote endpoints and storage documentation] for information about endpoints that are compatible with the remote write feature.
++
+[IMPORTANT]
+====
+Red{nbsp}Hat only provides information for configuring remote write senders and does not offer guidance on configuring receiver endpoints. Customers are responsible for setting up their own remote write compatible endpoints. Issues with endpoint receiver configurations are not included in Red{nbsp}Hat production support.
```

Comment:
just one concern, why it is `Red{nbsp}Hat`, and not `Red Hat`instead

Reply:
Hello, it is a part of our guidelines to add a non-breaking space  between  "Red" and "Hat". It is to prevent the company name from splitting across a line break. ([link to the guidelines](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces))
Thank you!

Reply:
got it, thanks!
/lgtm